src/r2x/upgrader/functions.py

```diff
@@ -427,7 +425,9 @@ def add_column(
         logger.debug(f"{fpath.name} has column etype already.")
         return None
 
-    data.insert(position, column, emission_source)
+    data.insert(
+        loc=1, column="etype", value="combustion"
```

Comment:
I am getting confused of this terminology. So, etype -> emission source and eall -> emission_type?